core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -81,3 +81,17 @@
     {% endif %}
   </ul>
 </script>
+<style type="text/css">
+.popover{
+  font-size: 13px;
+  max-width: 300px;
```

Comment:
Due to work on #3267 having this code, it might be best to remove it. As also mentioned in that PR, the scoping of this `.popover` may introduce errors elsewhere, so please scope it.